packages/apps/esm-primary-navigation-app/src/components/change-language/change-language.resource.ts

```diff
@@ -9,7 +9,7 @@ export type PostUserProperties = (
 
 export async function postUserPropertiesOnline(
   userUuid: string,
-  userProperties: any,
+  userProperties: Record<string, unknown>,
```

Comment:
In the backend it's just a map of strings...

```suggestion
  userProperties: Record<string, string>,
```